### ISSUE TITLE
🐛 BUG:  Client-side Application error

### ISSUE THREAD

**Issue by LarithuS** [dailydotdev/daily#1616]
### What went wrong? 🤔

Using chrome
When you disable companion and enable right after from customize 
"Application error: a client-side exception has occurred (see the browser console for more information)."
No issue after refreshing

### Steps to Reproduce Issue

```bash
1. Click on profile
3. Click on "Customize"
4. Disable "Enable Companion"
5. Enable "Enable Companion"
```

### Browsers

Chrome

### OS

Windows

### Additional Context

```bash
Failed to load resource: the server responded with a status of 404 ()Understand this errorAI
t1.gstatic.com/faviconV2?client=SOCIAL&type=FAVICON&fallback_opts=TYPE,SIZE,URL&url=https://gmail.com/&size=24:1 
        
        
       Failed to load resource: the server responded with a status of 404 ()Understand this errorAI
newtab.bundle.js?27f2bf459604570a039e:28 TypeError: a is not a function
    at newtab.bundle.js?27f2bf459604570a039e:72:227990
    at rl (newtab.bundle.js?27f2bf459604570a039e:28:93225)
    at Su (newtab.bundle.js?27f2bf459604570a039e:28:113355)
    at newtab.bundle.js?27f2bf459604570a039e:28:112097
    at xu (newtab.bundle.js?27f2bf459604570a039e:28:112162)
    at lu (newtab.bundle.js?27f2bf459604570a039e:28:105918)
    at Bo (newtab.bundle.js?27f2bf459604570a039e:28:46670)
    at cu (newtab.bundle.js?27f2bf459604570a039e:28:106220)
    at Te (newtab.bundle.js?27f2bf459604570a039e:28:15870)
    at $r (newtab.bundle.js?27f2bf459604570a039e:28:38524)
```

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by LarithuS** [comment]
I just started using it and I couldn't repeat the error after playing with some settings, I think I noticed it right after making my account,

**Comment by idoshamun** [comment]
So it's not reproducible at the moment?

**Comment by rebelchris** [comment]
Should be fixed in this one:
https://github.com/dailydotdev/apps/pull/3903

Thanks for reporting.

**Comment by rebelchris** [comment]
Closing, will be available in version: 3.35.5
